pipelines/maven-zip-build-oci-ta/README.md

```diff
@@ -1,165 +1,203 @@
 # "maven-zip-build-oci-ta pipeline"
-
```

Comment:
Commit message describes need of adding piplienes into main kustomization, why is this doc change required?

Please split README changes into separate commit and describe why change is done

Reply:
The README failed the CI checks so I added it here. I'll split them to another PR.

Reply:
The README fix is put in this PR: https://github.com/konflux-ci/build-definitions/pull/1692
After its merging I'll do rebase of this PR to it.

Reply:
Rebase done.

Reply:
thank you